config.php.default

```diff
@@ -49,3 +49,6 @@ $config['enable_billing'] = 1;
 
 # Enable the in-built services support (Nagios plugins)
 $config['show_services'] = 1;
+
+# Whether to use secure cookies. Should be true if you're using HTTPS
+$config['secure_cookies'] = false;
```

Comment:
Lets move this to `includes/defaults.inc.php` and then this looks good to go.

Thanks for discussing in depth :)